.ssh-keys/init-conf.json

```
@@ -0,0 +1,14 @@
+[
+    {"name":"s3","dns":"minio", "ns": "travis-TRAVIS_BUILD_NUMBER"},
```

I think it should be like that but I might be wrong.
```suggestion
    {"name":"s3","dns":"travis-minio-$TRAVIS_BUILD_NUMBER ", "ns": "travis-TRAVIS_BUILD_NUMBER"},
```